db/migrate/20220608144156_create_schemes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSchemes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :schemes do |t|
+      t.string :code
+      t.string :service_name
+      t.references :organisation, null: false, foreign_key: true
```

Comment:
I think elsewhere we've relied on application level `belongs_to` rather than database level foreign key constraint. This probably makes sense though... does a scheme have any meaning without an organisation?

Reply:
I am under impression that there isn't one without org, happy to amend if I am wrong.

Reply:
Should probably actually add this for users now too, and set delete to cascade

Reply:
With current spec - definitely cascade the whole lot, no point leaving ghosts behind